README.md

```diff
@@ -1,3 +1,33 @@
+
+![](https://github.com/NeuroDataDesign/the-ents/blob/explore-atlases/atlases/Results/brainAtlases_color_wRegions.png)
+
+
+## Atlas Info Summary
+
+| Atlas Name | # Regions | Symmetrical? | Hierarchical? | Labelled? | Generation Method | Average Vol/Region | Native coordinate space | Description | Sources | Year of Origin |
+|------------------------------|-----------|--------------|---------------|-----------|---------------------------------------------------------------------------------------------------------------------|--------------------|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|----------------|
+| Desikan | 70 | Yes | No | No | Anatomical Landmarks based on gyri. Averaged based on majority voting | 24786.857 |  | Surface parcellation | https://www.sciencedirect.com/science/article/pii/S1053811906000437?via%3Dihub | 2006 |
+| Glasser | 360 | Yes | Yes | Yes | Semi-automated. Separated based on function, connectivity, cortical architecture, topography, and expert analysis | 521.994 | MNI | Cortical parcellation from multi-modal images of 210 adults in HCP | https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127/ | 2016 |
+| AAL | 116 | No | No | Yes | Delineated with respect to anatomical landmarks (following sulci course in brain) | 12758.353 | MNI | Automated Anatomical Labelling | https://www.ncbi.nlm.nih.gov/pubmed/11771995 | 2002 |
+| CPAC200 | 200 | No |  | No |  | 5860.755 |  |  | https://github.com/FCP-INDI/C-PAC | 2018 |
+| HarvardOxford | 48 | No | Yes | Yes |  | 21966.104 |  |  | http://neuro.imm.dtu.dk/wiki/Harvard-Oxford_Atlas |  |
+| JHU | 48 | Yes | No | Yes | One subject manually labelled and warped to 29 other adult atlases (Large Deformation Diffeomorphic Metric Mapping) | 3541.792 |  | A small version of a larger (289 ROI) atlas composed based on parcellation of deep white matter. Split into 4 groups: Tracts in the brainstem, projection fibers, association fibers, and commisural fibers | https://www.ncbi.nlm.nih.gov/pubmed/14645885 | 2004 |
+| Talairach | 1105 | No | Yes | Yes | Semi-automated? | 1698.114 | Talairach coordinates | A hierarchical atlas split into 5 leves: Hemisphere, Lobe, Gyrus, Tissue Type, and Cell Type | https://www.ncbi.nlm.nih.gov/pubmed/7008525 | 1980 |
+| Brodmann | 40 | Yes | Yes | No | Brodman areas separated by gyri | 32978.512 |  | Corticall parcellation separating regions based on cellular morphology and organization | http://digital.zbmed.de/zbmed/id/554966 | 1909 |
+| Princeton | 49 | Yes | Yes | Yes |  | 1217.388 |  |  | https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4585523/ | 2015 |
+| pp264 | 58 | No |  | No |  | 470.966 |  |  |  |  |
+| Slab | 1068 |  |  | No |  | 493.719 |  |  |  |  |
```

Comment:
"Growth Charting Brain Connectivity Networks and the Identification of Attention Impairment in Youth" this parcellation I believe comes from here. https://www.nitrc.org/projects/kessler_jama16/ double check to be sure tho.